book/src/dev/rfcs/0010-v5-transaction.md

```diff
@@ -0,0 +1,245 @@
+- Feature Name: `v5_transaction`
+- Start Date: 2021-03-11
+- Design PR: [ZcashFoundation/zebra#1886](https://github.com/ZcashFoundation/zebra/pull/1886)
+- Zebra Issue: [ZcashFoundation/zebra#1863](https://github.com/ZcashFoundation/zebra/issues/1863)
+
+# Summary
+[summary]: #summary
+
+Network Upgrade number 5 (`NU5`) introduces a new transaction type (transaction version 5). This document is a proposed design for implementing such a transaction version.
+
+# Motivation
+[motivation]: #motivation
+
+The Zebra software wants to be a protocol compatible Zcash implementation. One of the tasks to do this includes the support of the new version 5 transactions that will be implemented in the next network upgrade.
```

Comment:
We should probably explicitly name "NU5" instead of referring to the "next network upgrade" since this is relative to the time this RFC is written

Reply:
```suggestion
The Zebra software wants to be a protocol compatible Zcash implementation. One of the tasks to do this includes the support of the new version 5 transactions that will be implemented in Network Upgrade 5 (NU5).
```